.github/dependabot.yml

```
@@ -3,10 +3,17 @@ updates:
-          interval: daily
-      open-pull-requests-limit: 10
+          interval: weekly
+          day: monday
+      open-pull-requests-limit: 5
       commit-message:
           prefix: "chore(deps)"
+      groups:
+          # Batch all minor/patch action bumps into a single PR to reduce noise.
+          # Major bumps still come through as individual PRs for review.
+          github-actions:
+              patterns: [ "*" ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)